### PR TITLE
fix: InitContainer logs not visible from app details page issue fix

### PR DESCRIPTION
**Pull request by sohel-rp**
# Description

These changes will fix InitContainer logs not visible issue for helm type apps.

Fixes - [AB#2270](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/2270)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

All tests have been performed manually by visiting AppDetails page for apps of type helm.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css [hunk 1]
```diff
@@ -16,4 +16,21 @@
 
 .summary-view__card {
     background-color: #f2f4f7;
+}
+
+.containers-select__group {
+    padding-top: 0 !important;
+    padding-bottom: 0 !important;
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css:23]: we can use .pb-0-imp

File: src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css [hunk 2]
```diff
@@ -16,4 +16,21 @@
...
+}
+
+.containers-select__group .containers-select__group-heading {
+    font-size: 12px;
+    font-weight: 600;
+    text-transform: none;
+    color: var(--N900);
```
Comment (vivek-devtron) [inline src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css:30]: can we minimize css?

File: src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css (hunk 1 above)
Comment (sohel-rp) [reply src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css:23]: Actually, this is written to apply internal styling to internal group class of react select with help of classNamePrefix.

File: src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css (hunk 2 above)
Comment (sohel-rp) [reply src/components/v2/appDetails/k8Resource/nodeDetail/nodeDetail.css:30]: Same here

**Review by vivek-devtron — APPROVED** [review]

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**